lib/draftjs_exporter/html.rb

```diff
@@ -66,8 +66,16 @@ def build_commands(block)
         Command.new(:start_text, 0),
         Command.new(:stop_text, block.fetch(:text).size)
       ] +
-        build_range_commands(:inline_style, :style, block.fetch(:inlineStyleRanges)) +
-        build_range_commands(:entity, :key, block.fetch(:entityRanges))
+      build_range_commands(
+        :inline_style,
+        :style,
+        block.fetch(:inlineStyleRanges) || []
```

Comment:
Is this for when the block doesn't have the inlineStyleRanges key defined?
What if `inlineStyleRanges == nil`? Do we want to return nil or an empty array?

Reply:
We want to return an empty array. Ruby doesn't have `undefined`, so this handles `block.fetch(:inlineStyleRanges)` returning `nil` for us or any falsy value.